internal/js/modules/k6/browser/common/browser_context.go

```diff
@@ -110,6 +110,16 @@ func (b *BrowserContext) setDownloadsPath(path string) error {
 	return nil
 }
 
+// cleanup cleans up the resources associated with the browser context.
+func (b *BrowserContext) cleanup() error {
```

Comment:
Do we want to cleanup a remote browser's browserContext's directory too? At the moment we only cleanup the data directory when we [start](https://github.com/grafana/k6/blob/7129195a85e57533564e46086018e43915d27945/internal/js/modules/k6/browser/common/browser_process.go#L186) a local chrome process.

Reply:
In this PR, we only clean up what we have directly created that is local to a k6 process. For remote browsers, `cleanup` is similar to a no-op:

> // If the path does not exist, RemoveAll
> // returns nil (no error).
https://pkg.go.dev/os#RemoveAll

Do you mean something else?

Reply:
Ah, right! It's downloaded files 👍